llvm/test/lab2/kuznetsov_artyom/test.ll

```diff
@@ -0,0 +1,75 @@
+; RUN: opt -passes=instr_func -S %s | FileCheck %s
```

Comment:
Could you please load this pass from dynamic library using `opt` option `load-pass-plugin`?